checks.go

```diff
@@ -0,0 +1,24 @@
+package datadog
+
+type Check struct {
+	Check     string   `json:"check"`
+	HostName  string   `json:"host_name"`
+	Status    status   `json:"status"`
+	Timestamp string   `json:"timestamp,omitempty"`
+	Message   string   `json:"message,omitempty"`
+	Tags      []string `json:"tags,omitempty"`
+}
+
+type status int
+
+const OK status = 0
```

Comment:
A more idiomatic way of representing this enum-like set of constants would be something like 

```go
const (
	OK status = iota
	WARNING
	CRITICAL
	UNKNOWN
)
```

see https://golang.org/doc/effective_go.html#constants for more information.